### DISCUSSION TITLE
多微软账号保存登陆(看到重复贴但不是很令我满意)

### DISCUSSION THREAD

**Discussion by SkyFutu** [Meloong-Git/PCL#496] [提问] [answered]
希望PCL2能做个多账号登陆，主要是我看到了个帖子说还要删除mojang登陆，这对我这个怀旧的人不是很友好，我建议可以看看隔壁HMCL做的多账号登陆，这样有多个账号的人就不用每次都要退掉重新登陆了，然后可以优化下登陆方式不需要内置跳转...

**Comment by WForst-Breeze** [comment, marked as answer]
您好，很高兴为您解答\(￣︶￣*\))
1. mojang 登录已经过时了，除了那些不怎么玩mc的，我想大部分玩家已经迁移玩账号了。（怀旧貌似没有实质性用处，启动器还是以好用、干净为主吧个人认为。但是暂时不去掉我认为还是可以的
2. 多账号登陆，请见 #43 。我相信这个功能做好后会跟隔壁启动器使用体验差不多的~
3. 优化登录方式，请见 #127

**Reply by SkyFutu** [reply]
那关于多账户登陆的项目要多久可以实现呢？每次退出在重新登陆实在难受

**Reply by WForst-Breeze** [reply]
> 那关于多账户登陆的项目要多久可以实现呢？每次退出在重新登陆实在难受

这我也不清楚，我也曾经问过龙猫（几个月之前吧），应该快了，再耐心等待等待吧~如果很着急用的话可以暂时先用隔壁启动器。我就是这样的，大号用PCL2，其他小号们就去隔壁启动器登录。🤔

**Reply by SkyFutu** [reply]
az好吧主要是隔壁启动器有点路径问题...或者有大佬直接实现发出来让龙猫c+v🤔

**Comment by Broken-Deer** [comment]
想法很好，但是，那不叫HTML谢谢

**Reply by SkyFutu** [reply]
啊啊啊啊啊我傻了写HTML写多了导致跟HMCL搞混了(笑哭)
